### PR TITLE
Python method for number of triangulation cells

### PR DESCRIPTION
**Pull request by agrayver**
Currently rebased against #10875, hence only last commit is relevant. 

Part of #9015

### CODE REVIEW

File: contrib/python-bindings/source/triangulation_wrapper.cc [hunk 1]
```diff
@@ -724,6 +724,24 @@ namespace python
...
 
+    template <int dim, int spacedim>
+    boost::python::list
+    cells(TriangulationWrapper &triangulation_wrapper)
+    {
+      Triangulation<dim, spacedim> *tria =
+        static_cast<Triangulation<dim, spacedim> *>(
+          triangulation_wrapper.get_triangulation());
+      boost::python::list cells_list;
+      for (auto &cell : tria->cell_iterators())
```
Comment (Rombur) [inline contrib/python-bindings/source/triangulation_wrapper.cc:735]:
```suggestion
      for (const auto &cell : tria->cell_iterators())
```

**Review by Rombur — APPROVED** [review]
Looks good

**Comment by Rombur** [comment]
clang-format is failing

**Comment by agrayver** [comment]
> clang-format is failing

Should be fixed now.

**Pull request merged**